index.html

```diff
@@ -24,7 +24,7 @@
 				<li><a href="#getting-started">Getting Started</a></li>
 				<li><a href="#examples">Examples</a></li>
 				<li><a href="#api">API</a></li>
-				<li><a href="#support">Browser support</a></li>
+				<li><a href="#support">Browser Support</a></li>
```

Comment:
Hi, @mike-engel. Thanks for the feedback. My thinking was to make the capitalization consistent with "Getting Started". I figured, perhaps incorrectly, that the intention had been to follow that common capitalization style for nav text, rather than changing "Getting Started" to "Getting started". It also happens to match the capitalization of the h2s for those sections further down the page.